src/main/java/org/broadinstitute/hellbender/utils/variant/writers/GVCFWriter.java

```diff
@@ -71,15 +73,20 @@ public GVCFWriter(final VariantContextWriter underlyingWriter, final List<Intege
     @VisibleForTesting
     static RangeMap<Integer,Range<Integer>> parsePartitions(final List<Integer> gqPartitions) {
         Utils.nonEmpty(gqPartitions);
-        Utils.containsNoNull(gqPartitions, "gqPartitions contains a null Integer");
+        Utils.containsNoNull(gqPartitions, "GQ partitions contains a null Integer");
         final RangeMap<Integer, Range<Integer>> result = TreeRangeMap.create();
         int lastThreshold = 0;
         for (final Integer value : gqPartitions) {
-            if (value < lastThreshold) {
-                throw new IllegalArgumentException("gqPartitions is out of order.  Last is " + lastThreshold + " but next is " + value);
+            if (value < 0) {
```

Comment:
Done. The first comment is too inefficient due to extra looping. The second comment is not applicable to the current code.